packages/storybook/package.json

```diff
@@ -18,16 +18,16 @@
   },
   "scripts": {
     "clean": "rimraf dist/ src/generated/",
-    "prebuild": "npm run clean",
+    "prebuild": "pnpm run clean",
```

Comment:
`pnpm` doesn't support `pre` and `post` scripts by default (for good reason), so I suggest we remove them and explicitly make them part of our `build` script.